slack/greeting

```diff
@@ -1,19 +1,19 @@
-Hey {{Joiner}}, Welcome to the JSON Schema Slack Server!
+:wave: Hey {{eec4a322-082c-4247-8cdd-b67d5eeee6aa==user}}, Welcome to the JSON Schema Slack Server!
 
 Whatever the reason you chose to join us, we want you to know, we're glad you're here.
 
-We have a formal [Code of Conduct](https://github.com/json-schema-org/.github/blob/main/CODE_OF_CONDUCT.md) which we invite you to review. It is short, and reading it is respectful to the community you've chosen to enter.
+We have a formal Code of Conduct which we invite you to review. It is short, and reading it is respectful to the community you've chosen to enter.
```

Comment:
Doesn't this remove the link to the Code of Conduct?

Reply:
This was a copy/paste issue when copying from Workflow builder. Fixed.